src/java/relex/ServerSession.java

```diff
@@ -180,15 +183,15 @@ public void handle_session(Socket in_sock, PrintWriter out)
 					// Print the phrase string ... handy for debugging.
 					out.println("; " + parse.getPhraseString());
 
-					if (verbose)
+					if (logger.isDebugEnabled())
 					{
 						String fin = SimpleView.printRelationsAlt(parse);
-						System.out.print(fin);
+						logger.debug(fin);
 					}
 					opencog.setParse(parse);
 					out.println(opencog.toString());
 					out.flush();
-					System.err.println("Info: hndlr=" + id + " sent parse " + (pn + 1) + " of " + np);
+					logger.info("Info: hndlr=" + id + " sent parse " + (pn + 1) + " of " + np);
```

Comment:
Why not use placeholders here as well?

    logger.info("Info: hndlr= {} sent parse {} of {}", id, pn + 1, np);

Reply:
fixed.